source/install/config-cloudfront.rst

```diff
@@ -0,0 +1,40 @@
+.. _config-cloudfront:
+
+Configuring CloudFront to host Mattermost static assets
+=======================================================
+
+1. Create an S3 bucket using your desired domain. In our example it will be mattermost.example.com.
+2. Enable static hosting for you S3 bucket.
+3. From the Mattermost distribution, upload the ``client`` directory to s3. Renaming it to ``static``. You can use aws CLI command below from within the client directory. Be sure however you do this that the files are public readable.
```

Comment:
what's the purpose of? `You can use aws CLI command below from within the client directory. Be sure however you do this that the files are public readable.`

Reply:
It's just an example of how to upload all the files to s3. And the second sentence is you need to make sure that the files have the permission public-read (the `--acl public-read` part of the command)